osm2gtfs/tests/tests_accra.py

```diff
@@ -158,9 +158,14 @@ def test_gtfs_from_cache(self):
                          "Wrong routes count in the generated GTFS")
 
 
-def suite():
-    tests = ['test_refresh_routes_cache', 'test_refresh_stops_cache', 'test_gtfs_from_cache']
-    return unittest.TestSuite(map(TestAccra, tests))
+def load_tests(loader, tests, pattern):
+    # pylint require to use the variables
+    loader = loader
+    tests = tests
+    pattern = pattern
```

Comment:
You should be able to do something like `# pylint: disable=unused-varialbes` here instead.

Reply:
I'm wondering what are the variables `loader`, `tests` and `pattern` used for? Do we need them? (sorry might be an ignorant question)

Reply:
I think they are just used here to make pylint happy which can also be achieved with something similar to what I proposed above.

Reply:
Ah, they are required by the discovery call. Thanks.

Reply:
@xamanu : variables are described here (and are used by the discovery) : https://docs.python.org/2/library/unittest.html#load-tests-protocol
@grote : thanks for the proposal, I'll check it out asap :)